backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -235,6 +235,46 @@ class SchedulingService implements ISchedulingService {
     return schedulingDtos;
   }
 
+  async getSchedulingsByPickUp(isPickUp: boolean): Promise<SchedulingDTO[]> {
+    let schedulingDtos: Array<SchedulingDTO> = [];
+    try {
+      const schedulings: Array<Scheduling> = await Scheduling.findAll({
+        where: { is_pickup: isPickUp, volunteer_needed: true },
```

Comment:
hmm don't we not need to query for `volunteer_needed` to be true here since as long as `is_pickup` is non-null, volunteer needed is _always_ true?

Reply:
I think isPickup is set as false when volunteerNeeded is false. In that case, if we don't query for volunteer needed is true, it returns all schedules that have ispickup is false even if it doesn't require volunteers.
<img width="377" alt="Screen Shot 2022-02-20 at 8 10 49 PM" src="https://user-images.githubusercontent.com/53016529/154873612-2c582d8d-ee0b-4c76-8cc1-62a4f5ff16ab.png">

Reply:
@hanlinc27 ^

Reply:
ohh you're right my bad! that looks good then 👍🏻